### PR TITLE
Removed transfer_native from erc20 frame

### PR DESCRIPTION
**Pull request by karthikeyan-cere**
Removed transfer_native function from erc20 from frame

### CODE REVIEW

File: frame/erc20/src/lib.rs [hunk 1]
```diff
@@ -81,20 +81,20 @@ decl_module! {
...
-            T::Currency::transfer(&source, &bridge_id, amount.into(), AllowDeath)?;
-
-            let resource_id = T::NativeTokenId::get();
-			let number_amount: u128 = amount.saturated_into();
-
-			// <bridge::Module<T>>::transfer_fungible(dest_id, resource_id, recipient, U256::from(amount.saturated_into()))
-            <bridge::Module<T>>::transfer_fungible(dest_id, resource_id, recipient, U256::from(number_amount))
-			// <bridge::Module<T>>::transfer_fungible(dest_id, resource_id, recipient, U256::from(UniqueSaturatedInto::<u128>::unique_saturated_into(amount)))
-        }
+        // #[weight = 195_000_000]
```
Comment (AndreiNavoichyk) [inline frame/erc20/src/lib.rs]: Please, use [block comment](https://doc.rust-lang.org/reference/comments.html).

File: frame/erc20/src/tests.rs [hunk 2]
```diff
@@ -48,31 +48,31 @@ fn transfer_hash() {
...
-        expect_event(bridge::RawEvent::FungibleTransfer(
-            dest_chain,
-            1,
-            resource_id,
-            amount.into(),
-            recipient,
-        ));
-    })
-}
+// #[test]
```
Comment (AndreiNavoichyk) [inline frame/erc20/src/tests.rs]: Please, use [block comment](https://doc.rust-lang.org/reference/comments.html).

**Review by AndreiNavoichyk — APPROVED** [review]

File: frame/erc20/src/lib.rs (hunk 1 above)
Comment (karthikeyan-cere) [reply frame/erc20/src/lib.rs]: Updated to use block comment

File: frame/erc20/src/tests.rs (hunk 2 above)
Comment (karthikeyan-cere) [reply frame/erc20/src/tests.rs]: Updated to use block comment

**Pull request merged**